mutant/modules/sarscov2_report.py

```diff
@@ -128,6 +128,36 @@ def create_fohm_csv(self):
                     ]
                 )
 
+
+    def create_instrument_properties(self):
+        """Creates a summary file for FoHM for each region-lab-combination"""
```

Comment:
change to """Creates a properties file with instrument information"""" or similar.